lib/roo_on_rails/railties/http.rb

```diff
@@ -17,10 +17,12 @@ class HTTP < Rails::Railtie
           ::Rack::Timeout
         )
 
+        middleware_to_insert_before = Rails::VERSION::MAJOR < 4 ? ::ActionDispatch::Cookies : ::Rack::Head 
```

Comment:
I assume this works implicitly because api mode doesn't exist before version 5, and we don't care if we insert before `Rack::Head` in all our services? I guess my question is - is it better to do it this way, or check for and switch on the existence of `ActionDispatch::Cookies` explicitly? (it could be that there is no difference)